system/services/formbuilder/FormBuilderService.cfc

```diff
@@ -959,6 +959,46 @@ component {
 		}
 	}
 
+	public string function cloneForm(
+		  required string basedOnFormId
+		, required string name
+		, required string description
+	) {
+		var originalFormData = getForm( id=arguments.basedOnFormId );
+		var cloneFormData = structNew();
+
+		for( var column in originalFormData.columnList ) {
+			if( !listFindNoCase( "id,datecreated,datemodified,_version_is_draft,_version_has_drafts", column ) ) {
+				if( listFindNoCase( "name,description", column ) ) {
+					cloneFormData[ "#column#" ] = arguments[ "#column#" ];
```

Comment:
`cloneFormData[ "#column#" ]` should just be `cloneFormData[ column ]` (same for `arguments[ column ]). See also lines below.